src/sentry.ts

```diff
@@ -0,0 +1,15 @@
+import * as Sentry from "@sentry/browser";
+
+export const initialize = () => {
+  const { sentryDsn, sentryEnv } = window.REEARTH_CONFIG ?? {};
+  if (sentryDsn) {
+    Sentry.init({
+      dsn: sentryDsn,
+      environment: sentryEnv,
+    });
+  }
+};
+
+export const reportError = (error: string) => {
```

Comment:
```suggestion
export const reportError = (error: any) => {
```

Type of `error` should be `any` because `Sentry.captureException` accepts `exception` arg with `any`.

https://github.com/getsentry/sentry-javascript/blob/67b76854bc0cee526930780e1d22f67dec1ff2bf/packages/core/src/baseclient.ts#L100